lib/ReactViewModels/ViewState.ts

```diff
@@ -319,37 +318,15 @@ export default class ViewState {
     this.terria = terria;
 
     this._unsubscribeNotificationListener = terria.notification.addEventListener(
```

Comment:
I think that if we're refactoring `Notification` stuff, we should probably get rid of the un-typed Cesium event handler in 

https://github.com/TerriaJS/terriajs/blob/20304b657d5e96766009f770a0c985f2345f8264/lib/Models/Terria.ts#L258

I know that `notificationState` should probably be in `ViewState`, but it needs to be visible to the entire codebase (especially the model layer). Could we maybe move it into `Terria.ts` and remove it from `ViewState`? As everywhere has access to the `terria` object, and then we can get rid of the `CesiumEvent` passing `Notifications` between `Terria` and `ViewState`.

Reply:
This approach could also be used for the `terria.error` Cesium event handler

Reply:
Never mind, not needed right now